declarations.d.ts

```diff
@@ -1246,7 +1246,11 @@ interface IProfileDir {
 	profileDir: string;
 }
 
-interface ICommonOptions extends IRelease, IDeviceIdentifier, IJustLaunch, IAvd, IAvailableDevices, IProfileDir {
+interface IEmulator {
```

Comment:
IEmulator -> IHasEmulator?

Reply:
I've moved the interface from NativeScript CLI. The idea of such interfaces is to wrap the properties used in `$options` (the ones that you can pass with `--` on the terminal).In this case, the option the user can pass is `--emulator`, so the interface became `IEmulator`.

Ping @KristianDD , @Fatme  - what do you think - should we change this to IHasEmulator (without changing the `--` option.

Reply:
IEmulatorTarget also sounds good to me

Reply:
What I mean is that IEmulator should be an interface for something that wanna call itself an Emulator. Here we need an interface for something that just has an emulator property => we need something like IHasEmulator or IHasEmulatorOption/IOptionsWithEmulator if we wanna specify that these interfaces are wrapping some options values.